backend/Zeus.Api.Domain/Common/ValueObjects/Authentication/AccessToken.cs

```diff
@@ -0,0 +1,8 @@
+namespace Zeus.Api.Domain.Common.ValueObjects.Authentication;
```

Comment:
Why put it in `Common/ValueObjects/Authentication` ? I think `Authentication/ValueObjects`  should be more adapted ?